src/monaca.js

```diff
@@ -2119,6 +2119,29 @@
     return deferred.promise;
   };
 
+  Monaca.prototype.jsStringEscape = function (string) {
+    return ('' + string).replace(/["'\\\n\r\u2028\u2029]/g, function (character) {
+      // Escape all characters not included in SingleStringCharacters and
+      // DoubleStringCharacters on
+      // http://www.ecma-international.org/ecma-262/5.1/#sec-7.8.4
+      switch (character) {
+        case '"':
+        case "'":
+        case '\\':
+          return '\\' + character
```

Comment:
Maybe adding `;` keeps our code consistent.
Also, this function don't necessary to be a public API in my point of view.